molgenis-biobank-plots/src/main/frontend/biobank-plots/src/components/Filters.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <form>
+    <h1>Filter</h1>
+    <h4>Data types</h4>
+    <div class="custom-controls-stacked">
```

Comment:
Don't forget you have the awesome power of bootstrap4 containers, containers-fluid etc..